connectors/src/connectors/google_drive/index.ts

```diff
@@ -597,6 +572,10 @@ export async function setGoogleDriveConnectorPermissions(
     }
   }
 
+  const webhooksRes = await registerWebhooksForAllDrives(connector, 0);
```

Comment:
Subscribing to the webhooks for each Google Drives we are syncing.

Reply:
Sure, but when I do that, other member of the team say "no object, only when parameters are subject to conflicting types", so I am a bit tired of changing depending of the reviewer, I guess I will stick to this schema from now on 👍 

(not directed towards you, but towards the team :))

Reply:
No worries it was a suggestion because when we read the code we wonder what it the 0 but that's definitely not a blocker!

Reply:
We standardise on objects most of the time except for specific situations (eg auth that is passed as first argument + object as second argument)

meta: complaining is not productive 👍